CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 ### General
 
 - Throw `IndexError` when using negative indexes on `PaginatedList` objects (Thanks, [@UniversalSuperBox](https://github.com/UniversalSuperBox))
+- The `overrides` attribute of `Assignments` now creates `AssignmentOverride` objects.
```

Comment:
Can we update this text to "`Assignment.overrides` now returns a list of `AssignmentOverride` objects."

Reply:
Done.